upgrades/upgrade_system.gd

```diff
@@ -47,8 +47,11 @@ func attempt_upgrade(upgrade: Upgrade) -> void:
 	_purse.spend_coins(cost)
 	upgrade.level_increased.emit()
 
+	SignalBus.upgrade_completed.emit()
```

Comment:
Would this be better named `upgrade_purchased`?

Reply:
idk i was trying to
differentiate "attempt upgrade" vs "upgrade succeeded"